experimental/packages/opentelemetry-shim-opencensus/package.json

```diff
@@ -64,7 +64,7 @@
     "@opentelemetry/core": "^1.0.0",
     "@opentelemetry/context-async-hooks": "^1.0.0",
     "semver": "^7.3.5",
-    "require-in-the-middle": "^6.0.0"
+    "require-in-the-middle": "^7.0.0"
```

Comment:
Not sure why this is needed here but better keep it up to date.

Reply:
It's not yet needed, but it will be part of a coming PR: https://github.com/open-telemetry/opentelemetry-js/pull/3778/files#diff-d3275c2b279bb6ca15b39ba62edeccb23c948c04c421f2083a4004493874122dR18

Reply:
ok, added some comments there. using RITM directly seems not that good to me thinking about duplicated versions, introduce IITM for ESM,...